src/stormpath.registration.js

```diff
@@ -85,7 +89,14 @@ angular.module('stormpath')
  * @param {string} postLoginState
  *
  * If using the `autoLogin` option, you can specify the name of a UI state that the user
- * should be redirected to after they successfully have registered.
+ * should be redirected to after they successfully have registered.  This is a UI Router
+ * integration, and requires that module.
+ *
+ * @param {string} postLoginPath
+ *
+ * If using the `autoLogin` option, you can specify the path that the user
+ * should be sent to after registration.  This value is passed to
+ * `$location.path()` and does not require a specific routing module.
```

Comment:
> ...and does not require a specific routing module.

Actually it requires the `ngRoute` module which is now its own module: <https://docs.angularjs.org/api/ngRoute>